src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
with current implementation you instantiate `class Random` each time method is invoked.

you should instantiate this class one time and use it multiple times.

Reply:
Could you please clarify what you mean? I implemented this based on the examples provided in the theory section of the OOP module on the Mate Academy platform. This is the approach I was taught by your platform.

I am certainly open to acquiring new knowledge and would greatly appreciate it if you could provide me with an example of a different implementation.
<img width="846" alt="Screenshot 2023-09-02 at 20 44 28" src="https://github.com/mate-academy/jv-lottery/assets/141838187/695ddc41-2566-4566-a947-2bc10f573d0c">